packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -228,20 +233,22 @@ class BottomSheetRangeCell extends Component {
 						style={ styles.slider }
 						accessibilityRole={ 'adjustable' }
 					/>
-					<TextInput
-						style={ [
-							defaultSliderStyle,
-							borderStyles.borderStyle,
-							hasFocus && borderStyles.isSelected,
-							{ width: 40 * fontScale },
-						] }
-						onChangeText={ this.handleChange }
-						onFocus={ this.handleToggleFocus }
-						onBlur={ this.handleToggleFocus }
-						keyboardType="numeric"
-						returnKeyType="done"
-						value={ `${ sliderValue }` }
-					/>
+					{ withTextInput && (
```

Comment:
isn't 'withXXX' is a naming convention for HOCs? This looks like a boolean prop so, can we update the naming to reflect the intent more?

Reply:
That's true, I've renamed it to `shouldDisplayTextInput`

Reply:
Thanks!